node/src/chain_spec.rs

```diff
@@ -173,7 +173,62 @@ pub fn remote_testnet_config() -> Result<ChainSpec, String> {
 		Some("Golden Gate"),
 		None,
 		// Properties
-		properties(),
+		properties("GGXT"),
```

Comment:
@akorchyn Can we make a better sounding name here like "GGX Testnet"?

Reply:
It's token symbol, we decided that it will be ggxt